src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,55 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import { ChangeEvent } from 'react';
+import cn from 'classnames';
+
+import { centuries } from '../constants/centuries';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sexFronParams = searchParams.get('sex');
+  const centuriesFromParams = searchParams.getAll('century');
+  const queryFromParams = searchParams.get('query') || '';
+
+  function handleChangeQuery(event: ChangeEvent<HTMLInputElement>) {
+    setSearchParams(
+      getSearchWith(searchParams, { query: event.target.value || null }),
+    );
+  }
+
+  function handleChangeCenturies(century: string) {
+    const currentCenturies = centuriesFromParams;
+
+    return currentCenturies.includes(century)
+      ? currentCenturies.filter(currCentury => currCentury !== century)
+      : [...currentCenturies, century];
+  }
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">
+        <Link
+          className={!sexFronParams ? 'is-active' : ''}
```

Comment:
Typo: The variable `sexFronParams` should be renamed to `sexFromParams` for consistency and clarity.